Client/src/components/contanier/Auth/index.js

```diff
@@ -1,27 +1,117 @@
-import React, { Component } from 'react'
+import React, {
+  Component
+} from 'react'
+import axios from 'axios';
 
 import Button from '../../common/Button'
 import LabeledInput from '../../common/LabeledInput'
 import logo from '../../assets/Image/amideasblue.png'
-import { firstLable, secondLable } from './staticdata'
 import './index.css'
 
 export default class Login extends Component {
 
+  state = {
+    id: '',
+    password: '',
+    loginError: '',
+    role: '',
+  };
+
+  handleChange = ({
+    target: {
+      name,
+      value
+    }
+  }) => {
+    this.setState({
+      [name]: value.trim()
+    });
+  };
+
   handleClick = (e) => {
-    e.preventdefault()
+    e.preventDefault()
+    const {
+      id,
+      password
+    } = this.state;
+    if (!id || !password) {
+      this.setState({
+        loginError: 'All Fields Requier'
+      })
+    } else if (id.length === 0) {
+      this.setState({
+        loginError: 'ID At least Five Numbers'
+      })
+    } else if (password.length < 5) {
+      this.setState({
+        loginError: 'Password At Least Five'
+      })
+    } else {
+      this.setState({
+        loginError: ''
+      });
+      axios
+        .post('/api/v1/login', {
+          id,
+          password
+        })
+        .then(res => {
```

Comment:
```suggestion
  .then(res => {
     const { error } = res.data;
     const { data } = res.data; 
```
**just to be more readable**